core/src/main/scala/cats/Eval.scala

```diff
@@ -100,7 +109,9 @@ sealed abstract class Eval[A] { self =>
  * This type should be used when an A value is already in hand, or
  * when the computation to produce an A value is pure and very fast.
  */
-case class Now[A](value: A) extends Eval[A]
+case class Now[A](value: A) extends Eval[A] {
```

Comment:
Not part of your PR, but I just noticed that the `Eval` constructors should probably be final.